system-tests/test/minting-unminting.test.ts

```diff
@@ -326,21 +372,20 @@ describe("System Test - Minting and unminting", () => {
             )
 
             // Request redemption to depositor's address.
-            redeemerOutputScript = `0014${computeHash160(
-              systemTestsContext.depositorBitcoinKeyPair.publicKey.compressed
-            )}`
+            redeemerOutputScript = Hex.from(
+              `0014${BitcoinHashUtils.computeHash160(
+                systemTestsContext.depositorBitcoinKeyPair.publicKey.compressed
+              )}`
+            )
 
             // Convert the redeemer output script to a byte buffer.
-            const rawRedeemerOutputScript = Buffer.from(
-              redeemerOutputScript,
-              "hex"
-            )
+            const rawRedeemerOutputScript = redeemerOutputScript.toBuffer()
 
             const redemptionData = ethersUtils.defaultAbiCoder.encode(
               ["address", "bytes20", "bytes32", "uint32", "uint64", "bytes"],
               [
                 systemTestsContext.depositor.address,
-                `0x${deposit.walletPublicKeyHash}`,
+                depositReceipt.walletPublicKeyHash.toPrefixedString(),
```

Comment:
Unmint and redeem should be done using the new way: https://docs.threshold.network/app-development/tbtc-v2/tbtc-sdk/guides/unmint-and-redeem

Reply:
Done in 5691a18c4dff48f0fcfc198c32e245ed28903472.